axltempl/lando.py

```diff
@@ -56,6 +56,20 @@ def generateLandoFiles(name, docroot, cache):
         os.mkdir(".lando")
     util.copyPackageFile("files/lando/php.ini", ".lando/php.ini")
 
+    # Generate lando development override configuration.
+    generateLandoDevelopmentSettingsFiles(docroot, cache)
```

Comment:
@shinde-rahul, any value in this new function? I mean, is there a scenario where we would just call this function and not `generateLandoFiles`?

Reply:
Merging both the functions. This function was called from drupal.py, which is not the case now.